packages/bot-web-ui/src/pages/bot-builder/quick-strategy/quick-strategy.tsx

```diff
@@ -47,27 +64,13 @@ const FormikWrapper: React.FC<TFormikWrapper> = observer(({ children }) => {
         current_duration_min_max,
         initializeLossThresholdWarningData,
     } = quick_strategy;
+    const { active_tab } = dashboard;
+    const { setValueServerBot, createBot, setFormValues } = server_bot;
     const config: TConfigItem[][] = STRATEGIES[selected_strategy]?.fields;
     const [dynamic_schema, setDynamicSchema] = useState(Yup.object().shape({}));
 
-    const initial_value: TFormData = {
-        symbol: qs_config.QUICK_STRATEGY.DEFAULT.symbol,
-        tradetype: '',
-        durationtype: qs_config.QUICK_STRATEGY.DEFAULT.durationtype,
-        stake: '1',
-        loss: '0',
-        profit: '0',
-        size: String(qs_config.QUICK_STRATEGY.DEFAULT.size),
-        duration: '1',
-        unit: String(qs_config.QUICK_STRATEGY.DEFAULT.unit),
-        action: 'RUN',
-        max_stake: 10,
-        boolean_max_stake: false,
-        last_digit_prediction: 1,
-    };
-
     React.useEffect(() => {
-        const data = JSON.parse(localStorage.getItem('qs-fields') || '{}');
+        const data = JSON.parse(localStorage.getItem(active_tab === 4 ? 'server-form-fields' : 'qs-fields') || '{}');
```

Comment:
```suggestion
        const data = JSON.parse(localStorage.getItem(active_tab === SERVER_SIDE_BOT_TAB_NO ? 'server-form-fields' : 'qs-fields') || '{}');
```
is it possible to use enums instead of this number? it makes the code much more readable!